tests/acceptance/tests/MigrationByUiFlow.spec.ts

```diff
@@ -102,17 +102,6 @@ test('As a shop owner I want to migrate my data from my old SW5 shop to SW6 via
         await page.getByRole('button', { name: 'Back to overview' }).click();
     });
 
-    // ToDo MIG-985: Remove this if the underlying issue is fixed
-    await test.step('Wait for media download to finish', async () => {
-        await expect.poll(async () => {
-            return await MediaProcessObserver.isMediaProcessing();
```

Comment:
I guess this was the only place where `MediaProcessObserver` was used? So you can remove that file as well:
https://github.com/shopware/SwagMigrationAssistant/blob/17155df474c74cabd3a70aa8f5185ca4e312580f/tests/acceptance/fixtures/MediaProcessObserver.ts